doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-con.adoc

```diff
@@ -0,0 +1,77 @@
+[id='knowledge-store-rest-api-con_{context}']
+= Knowledge Store REST API for {CENTRAL} spaces and projects
+
+{PRODUCT} provides a Knowledge Store REST API that you can use to interact with your projects and spaces in {PRODUCT} without using {CENTRAL}. The Knowledge Store is the artifact repository for assets in {PRODUCT}. This API support enables you to maintain your {PRODUCT} resources more efficiently and optimize your integration and development with {PRODUCT}.
+
+With the Knowledge Store REST API, you can perform the following actions:
+
+* Retrieve information about all projects and spaces
+* Create, update, or delete projects and spaces
+* Build, deploy, and test projects
+* Retrieve information about previous Knowledge Store REST API requests, or _jobs_
+
+Knowledge Store REST API requests require the following components:
+
+Authentication::
+The Knowledge Store REST API requires basic HTTP Authentication or token-based authentication for the user role `rest-all`. To view configured user roles for your {PRODUCT} distribution, navigate to `~/$SERVER_HOME/standalone/configuration/application-roles.properties` and `../application-users.properties`.
++
+--
+To add a user with the `rest-all` role, navigate to `~/$SERVER_HOME/bin` and run the following command:
+
+[source,bash]
+----
+$ ./add-user.sh -a --user <USERNAME> --password <PASSWORD> --role rest-all
+----
+
+For more information about user roles and {PRODUCT} installation options, see
+ifdef::PAM,DM[]
+{URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
+endif::[]
+ifdef::DROOLS,JBPM[]
+<<_installing_the_kie_server>>.
+endif::[]
+--
+
+HTTP headers::
+The Knowledge Store REST API requires the following HTTP headers for API requests:
++
+* `Accept`: Data format accepted by your requesting client:
+** `application/json` (JSON)
+* `Content-Type`: Data format of your `POST` or `PUT` API request data:
+** `application/json` (JSON)
+
+HTTP methods::
+The Knowledge Store REST API supports the following HTTP methods for API requests:
```

Comment:
I don't get why add this here, I think it makes more sense to mention the headers on individual bases.

btw: IIRC the name of this is HTTP verbs, not methods

Reply:
This section in general follows a consistent pattern for all three APIs, and is a commonly provided piece of info for API documents. I'm just trying to be consistent with standard API docs and best practices.

From what I've seen in working with API docs, "methods" is the proper and more common term for it, while "verbs" is more informal. Google "HTTP methods vs verbs", for example.

Reply:
k